src/components/MessageCard/MessageCard.jsx

```diff
@@ -129,6 +133,7 @@ MessageCard.defaultProps = {
   in: true,
   isMobile: false,
   isWithBoxShadow: true,
+  onBodyClick: () => {},
```

Comment:
```suggestion
  onBodyClick: noop,
```

I would replace the arrow function with the noop variable (it look better in the docs)